packages/mint-components/src/components/tax-and-cash/sqm-payout-status-alert/sqm-payout-status-alert.feature

```diff
@@ -0,0 +1,60 @@
+@author:andy @owner:andy
+Feature: Cash payout status widget alert
+
```

Comment:
The identity verification state should be added to the alert built into the cash component as well

Reply:
👍🏻